src/website/app/demos/Popover/bestPractices.js

```diff
@@ -0,0 +1,114 @@
+    super(props);
+    this.state = { open: false };
+  }
+
+  render() {
+    const DemoContent = (
+      <GoodDemo>
+        <p>
+          Malachite often results from the weathering of copper ores, and is
+          often found together with azurite (Cu₃(CO₃)₂(OH)₂), goethite, and
+          calcite. Malachite is more common than azurite and is typically
+          associated with copper deposits around limestones, the source of the
+          carbonate.
+        </p>
+      </GoodDemo>
+    );
+    return (
+      <DemoLayout>
+        <Button
+          primary
+          onClick={() => this.setState({ open: !this.state.open })}>
+          Mine ore
+        </Button>
+        <Popover isOpen={this.state.open} content={DemoContent}>
+          <P>Malachite</P>
+        </Popover>
+      </DemoLayout>
+    );
+  }
+}
+
+const BadContent = (
+  <div>
+    <Heading level={2}>Crystallography</Heading>
+    <p>
+      Kröhnkite has monoclinic symmetry (2/m).{' '}
+      <Link href="https://en.wikipedia.org/wiki/Monoclinic_symmetry">
+        Monoclinic symmetry
+      </Link>{' '}
+      implies that the mineral contains three axes of differing length
+      (typically labeled a, b, and c), two of which intersect each other at 90°
+      and one that intersects at an acute angle. Specifically, it belongs to the
+      2/m symmetry class meaning, the mineral has a 2-fold rotation axis about
+      the b axis. It also has a unique motif of silicon tetrahedra chains and
+      copper octahedra aligned along the c axis and linked together by sodium
+      atoms. Kröhnkite exhibits the optical property{' '}
+      <Link href="https://en.wikipedia.org/wiki/Birefringence">
+        birefringence
+      </Link>; the difference in the two refractive indices of a mineral.
+      Because this mineral is birefringent, it must be anisotropic. Anisotropic
+      minerals cause the velocity of light to vary depending on the direction of
+      travel through the mineral. Kröhnkite is biaxial negative, which reveals
+      that the mineral has two optic axes.
+    </p>
+    <Heading level={2}>Importance</Heading>
+    <p>
+      Kröhnkite has the same general formula (X₂M(TO₄)₂(H₂O)₂) as minerals which
+      are found in environments affected by hydrothermal alteration, making it
+      important in identifying where such alterations have occurred.
+      Furthermore, the minerals sharing this composition are organized according
+      to three crystal structure types, one being the unique kröhnkite structure
+      which is often used to describe minerals exhibiting the same chain-like
+      structure.
+    </p>
+  </div>
+);
+
+export default [
+  {
+    type: 'do',
+    title: 'use Popover to provide help to the user',
+    example: <PopOverDo />,
+    description: 'Provide extra context for selected interface elements.'
+  },
+  {
+    type: 'dont',
+    title: 'add too much content',
+    example: (
+      <Popover content={BadContent}>
```

Comment:
This Popover does not behave properly.  I understand that you are trying to show a bad example, but it should still behave properly.  Likely to do with needing content layout, perhaps it needs a max-width or something...

Reply:
I was trying to let consumers know that the popover will fully break when you put too much content without styles on it. Maybe I should call out a possible solution (like `max-width`) in the description?

Reply:
either way, it does look quite broken. I guess we have to figure out where the line of appearing "intentionally broken" lies.

Reply:
I'm going to go ahead and fix this; we can get the point across with a normal-sized popover with a long scroll.